server/app/auth/oidc/applicant/IdcsProvider.java

```diff
@@ -72,7 +72,7 @@ protected String getResponseType() {
     // https://github.com/panva/node-oidc-provider/blob/main/docs/README.md#responsetypes
     // TODO: turn this into it's own fake provider or make the local provider allow
     // 'token'.
-    if (baseUrl.contains("localhost:")) {
+    if (baseUrl.contains("localhost:") || baseUrl.startsWith("http://civiform:")) {
```

Comment:
can you explicitly comment what the two fake domains support?

Previously it's effectively all "local fake IDCS" but now a reader isn't going to know where the lines are in our testing between the names.

Reply:
Done.